odoo/tools/mail.py

```diff
@@ -4,6 +4,8 @@
 import base64
 import collections
 import logging
+
+import markupsafe
```

Comment:
HumanLinter is sad about your import oredering and spacing (unnecessary blank like before and missing one after)

Reply:
HumanLinter is wrong.

Reply:
Actually human linter is confused as generally odoo imports are not looking like standard lib imports. Sad humain linter.